src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -235,6 +235,23 @@
                 (read [_ rdr tag component-count]
                   (read-with-meta rdr #(apply list %))))}}
 
+   "clj/emptylist"
+   {:class clojure.lang.PersistentList$EmptyList
+    :writer (reify WriteHandler
+              (write [_ w o]
+                (let [m (meta o)]
+                  (do
+                    (.writeTag w "clj/emptylist" 2)
```

Comment:
Now that there's only one component written out this should be 1, not 2.  See the [Fressian custom handler documentation](https://github.com/clojure/data.fressian/wiki/Creating-custom-handlers); the implementation of this method in [FressianWriter](https://github.com/Datomic/fressian/blob/fressian-0.6.6/src/org/fressian/FressianWriter.java#L252) might be good to glance at as well.